src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -56,12 +64,17 @@ class ServiceTypeAhead extends Component {
   };
 
   render() {
-    const { services } = this.state;
+    const { defaultSelectedItem, services } = this.state;
     // eslint-disable-next-line prettier/prettier
     const renderService = (s) => { return (s && s.name) ? s.name.trim() : ''; };
 
     return (
-      <Downshift onChange={this.handleOnSelect} itemToString={renderService}>
+      <Downshift
+        onChange={this.handleOnSelect}
+        defaultSelectedItem={defaultSelectedItem}
```

Comment:
Was simple to find this once I realized we were two versions behind and I had to look at the [git history](https://github.com/paypal/downshift/blob/531bd8ea83ae5ecc3c163357ac9be4dc721d2060/README.md#defaultinputvalue) of Downshift's readme to find the right docs.